src/LiveComponent/src/Twig/LiveComponentRuntime.php

```diff
@@ -45,4 +47,25 @@ public function getComponentUrl(string $name, array $props = []): string
 
         return $this->urlGenerator->generate($metadata->get('route'), $params, $metadata->get('url_reference_type'));
     }
+
+    public function getLiveAction(string $actionName, array $parameters = [], array $modifiers = [], ?string $event = null): string
+    {
+        $attributes = $this->stimulusHelper->createStimulusAttributes();
+
+        $prevent = false;
+        if (\array_key_exists('prevent', $modifiers) && $modifiers['prevent']) {
+            $prevent = true;
+            unset($modifiers['prevent']);
+        }
```

Comment:
I'm thinkink we should maybe not deal with this and let the user type 'validate:prevent' for instance... 

because these are not modifiers for use like 'addClass' or 'delay' but Stimulus ones.. 

```twig
<button {{ live_action('save:prevent') }}>
```

wdyt ?